src/Kernel-Tests/MyVariableNotDeclaredDummy.class.st

```diff
@@ -0,0 +1,16 @@
+"
+For the Class part:  ""I represent a VariableNotDeclared with overriden methods for unit tests"".
+
+For the Responsibility part: Used for testing the base class.
+"
+Class {
+	#name : #MyVariableNotDeclaredDummy,
+	#superclass : #VariableNotDeclared,
+	#category : #'Kernel-Tests-Exceptions'
+}
+
+{ #category : #accessing }
+MyVariableNotDeclaredDummy >> messageBasedDescription [ 
+	"Overrides the method for testing"	
+	^ 'messageBasedDescription'
+]
```

Comment:
Same remark as for the MNU above

Reply:
The reason of the dummy was the same as the previous. Unfortunately, for this case, I was not able to generate a working "VariableNotDeclared" exception. Please take a look if the solution is acceptable.